### PR TITLE
Reviewer contribution role

### PR DESCRIPTION
**Pull request by hexylena**
another _fantastic_ community oriented feature brought to you by @nomadscientist asking nicely and me being employed to implement it™ 

The reviewers list gets quite long, so, we've chosen to just use their pfp instead. Reviewing is of course incredibly important, but, not as important as e.g. authorship. Currently this is automatically populated by the metadata/github.yml file, but you can add to it as well, if you want. Just set `reviewing: [your-reviewers]`.

Screenshot 2024-12-02 at 11-58-16 Hands-on A short introduction to Galaxy _ A short introduction to Galaxy _ Introduction to Galaxy Analyses

Screenshot 2024-12-02 at 12-22-49 Hands-on Use Apptainer containers for running Galaxy jobs _ Use Apptainer containers for running Galaxy jobs _ Galaxy Server administration

look at how many folks took the time to say something about that tutorial, but, weren't getting credit for it.

### CODE REVIEW

File: news/_posts/2024-12-02-reviewing.md
Comment (hexylena) [inline news/_posts/2024-12-02-reviewing.md:1]: @nomadscientist could you check how you feel about the contents of this file? Anything I should add or expand?

Comment (nomadscientist) [reply news/_posts/2024-12-02-reviewing.md:1]: Article looks ace

**Comment by hexylena** [comment]
> Question - so is the 'reviewers' automated and the manual?

yes, just add `reviewing: [nomadscientist]` (or so, to manually credit them.)

> Does it go in the same category? 

yes

> Or are there 2 reviewing categories? (I.e. we discussed preventing double entries of both manual and Github)

we did and I realised i didn't need to add that complexity so it's just one list, that's uniq'd at the end to ensure no duplicates.

**Review by shiltemann — APPROVED** [review]

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Awesome, thanks @hexylena and @nomadscientist for the idea :)

**Pull request merged**